flight/PiOS/STM32F4xx/pios_flash_internal.c

```diff
@@ -34,6 +36,9 @@
 #include "pios_semaphore.h"
 #include <stdbool.h>
 
+/* Todo: consider relocating to a good header */
+bool __attribute__((weak)) vehicle_is_armed = false;
```

Comment:
I looked at the pios_sys idea and it's not good because it's also not shared everywhere (would need to duplicate code between arches).  Putting it in pios_flash_internal isn't good because the code in manualcontrol would need two ifdefs-- F4 and PIOS_INCLUDE_FLASH_INTERNAL which IMO is destroying a lot of abstraction.

Can we defer?  I think we need a new module to keep track of state like this, and I feel like we need to get into shipping cadence soon.